docs/bigquery-usage.rst

```diff
@@ -236,3 +236,189 @@ Delete a table:
    >>> dataset = client.dataset('dataset_name')
    >>> table = dataset.table(name='person_ages')
    >>> table.delete()  # API request
+
+Jobs
+----
+
+Jobs describe actions peformed on data in BigQuery tables:
+
+- Load data into a table
+- Run a query against data in one or more tables
+- Extrat data from a table
+- Copy a table
+
+List jobs for a project:
+
+.. doctest::
+
+   >>> from gcloud import bigquery
+   >>> client = bigquery.Client()
+   >>> jobs = client.jobs()  # API request
+   >>> [(job.job_id, job.type, job.created, job.state) for job in jobs]
+   ['e3344fba-09df-4ae0-8337-fddee34b3840', 'insert', (datetime.datetime(2015, 7, 23, 9, 30, 20, 268260, tzinfo=<UTC>), 'done')]
+
+Querying data (synchronous)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run a query which can be expected to complete within bounded time:
+
+.. doctest::
+
+   >>> from gcloud import bigquery
+   >>> client = bigquery.Client()
+   >>> query = """\
+   SELECT count(*) AS age_count FROM dataset_name.person_ages
```

Comment:
Do we want to build an API for this grammar or just let the user send text? (Similar to us not-supporting SQL in datastore.)